netstumbler.cc

```diff
@@ -24,8 +24,8 @@
 #include <cstdio>                  // for snprintf
 #include <cstdlib>                 // for atoi, atof, qsort, strtol
 #include <cstring>                 // for strcpy, strlen, memset, strncmp, strstr
-#include <ctime>                   // for mktime
 
+#include <QtCore/QDateTime>        // for QDateTime
```

Comment:
I am still fighting with include-what-you-use, but here is what it suggests with my current mapping file:

```
./iwyu netstumbler

netstumbler.cc should add these lines:
#include <QtCore/QDate>      // for QDate
#include <QtCore/QTime>      // for QTime
#include <QtCore/QVector>    // for QVector
#include <QtCore/Qt>         // for UTC

netstumbler.cc should remove these lines:
- #include <QtCore/QtGlobal>  // lines 30-30

The full include-list for netstumbler.cc:
#include <QtCore/QDate>      // for QDate
#include <QtCore/QDateTime>  // for QDateTime
#include <QtCore/QString>    // for QString
#include <QtCore/QTime>      // for QTime
#include <QtCore/QVector>    // for QVector
#include <QtCore/Qt>         // for UTC
#include <cctype>            // for isspace
#include <cstdio>            // for snprintf
#include <cstdlib>           // for atof, atoi, qsort, strtol
#include <cstring>           // for strcpy, strlen, strncmp, strstr
#include "cet_util.h"        // for cet_convert_init
#include "csv_util.h"        // for csv_lineparse
#include "defs.h"            // for arglist_t, Waypoint, ff_cap, WaypointList, ff_cap_none, ddmm2degrees, ARG_NOMINMAX, ARGTYPE_STRING, ff_cap_read, get_crc32, lrtrim, waypt_add, xfree, xmalloc, ff_type_file, ARGTYPE_BOOL, CET_CHARSET_ASCII, CET_CHARSET_UTF8, CSTR, NULL_POS_OPS, ff_vecs_t
#include "gbfile.h"          // for gbfclose, gbfgetstr, gbfopen, gbfile
```

Reply:
Thanx. Incorporated.

Yesterday, I tried to resist the urge to "fix" several of these formats that should probably just be dropped. I largely failed and got sucked into maintenance on really old code.